src/gamecontroller.cs

```diff
@@ -535,4 +621,1162 @@ interface GCEventViewController {
+        NSString PageUp { get; }
+
+        [Field ("GCKeyDeleteForward")]
+        NSString DeleteForward { get; }
+
+        [Field ("GCKeyEnd")]
+        NSString End { get; }
+
+        [Field ("GCKeyPageDown")]
+        NSString PageDown { get; }
+
+        [Field ("GCKeyRightArrow")]
+        NSString RightArrow { get; }
+
+        [Field ("GCKeyLeftArrow")]
+        NSString LeftArrow { get; }
+
+        [Field ("GCKeyDownArrow")]
+        NSString DownArrow { get; }
+
+        [Field ("GCKeyUpArrow")]
+        NSString UpArrow { get; }
+
+        [Field ("GCKeyKeypadNumLock")]
+        NSString KeypadNumLock { get; }
+
+        [Field ("GCKeyKeypadSlash")]
+        NSString KeypadSlash { get; }
+
+        [Field ("GCKeyKeypadAsterisk")]
+        NSString KeypadAsterisk { get; }
+
+        [Field ("GCKeyKeypadHyphen")]
+        NSString KeypadHyphen { get; }
+
+        [Field ("GCKeyKeypadPlus")]
+        NSString KeypadPlus { get; }
+
+        [Field ("GCKeyKeypadEnter")]
+        NSString KeypadEnter { get; }
+
+        [Field ("GCKeyKeypad1")]
+        NSString Keypad1 { get; }
+
+        [Field ("GCKeyKeypad2")]
+        NSString Keypad2 { get; }
+
+        [Field ("GCKeyKeypad3")]
+        NSString Keypad3 { get; }
+
+        [Field ("GCKeyKeypad4")]
+        NSString Keypad4 { get; }
+
+        [Field ("GCKeyKeypad5")]
+        NSString Keypad5 { get; }
+
+        [Field ("GCKeyKeypad6")]
+        NSString Keypad6 { get; }
+
+        [Field ("GCKeyKeypad7")]
+        NSString Keypad7 { get; }
+
+        [Field ("GCKeyKeypad8")]
+        NSString Keypad8 { get; }
+
+        [Field ("GCKeyKeypad9")]
+        NSString Keypad9 { get; }
+
+        [Field ("GCKeyKeypad0")]
+        NSString Keypad0 { get; }
+
+        [Field ("GCKeyKeypadPeriod")]
+        NSString KeypadPeriod { get; }
+
+        [Field ("GCKeyKeypadEqualSign")]
+        NSString KeypadEqualSign { get; }
+
+        [Field ("GCKeyNonUSBackslash")]
+        NSString NonUSBackslash { get; }
+
+        [Field ("GCKeyApplication")]
+        NSString Application { get; }
+
+        [Field ("GCKeyPower")]
+        NSString Power { get; }
+
+        [Field ("GCKeyInternational1")]
+        NSString International1 { get; }
+
+        [Field ("GCKeyInternational2")]
+        NSString International2 { get; }
+
+        [Field ("GCKeyInternational3")]
+        NSString International3 { get; }
+
+        [Field ("GCKeyInternational4")]
+        NSString International4 { get; }
+
+        [Field ("GCKeyInternational5")]
+        NSString International5 { get; }
+
+        [Field ("GCKeyInternational6")]
+        NSString International6 { get; }
+
+        [Field ("GCKeyInternational7")]
+        NSString International7 { get; }
+
+        [Field ("GCKeyInternational8")]
+        NSString International8 { get; }
+
+        [Field ("GCKeyInternational9")]
+        NSString International9 { get; }
+
+        [Field ("GCKeyLANG1")]
+        NSString LANG1 { get; }
```

Comment:
```suggestion
        NSString Lang1 { get; }
```

and the same for all the other LANG ones.